scripts/pr_previews/utils.py

```diff
@@ -28,6 +29,31 @@ def configure_logging() -> None:
     )
 
 
+def setup_git_account() -> None:
+    run_subprocess(["git", "config", "user.name", "github-actions[bot]"])
+    run_subprocess(
+        ["git", "config", "user.email", "github-actions[bot]@users.noreply.github.com"]
+    )
+
+
+def changed_files() -> None:
+    return run_subprocess(["git", "status", "--porcelain"]).stdout.strip()
+
+
+def commit_all_and_push(commit_message: str) -> None:
+    run_subprocess(["git", "add", "."])
+    run_subprocess(["git", "commit", "-m", commit_message])
+    run_subprocess(["git", "push"])
+
+
+@contextmanager
+def switch_branch(branchname: str) -> None:
```

Comment:
I think you need to import from `typing`
```suggestion
def switch_branch(branchname: str) -> Iterator[None]:
```